docs/use-service.md

```diff
@@ -68,6 +74,100 @@ https://tile.mapzen.com/mapzen/terrain/v1/skadi/{N|S}{y}/{N|S}{y}{E|W}{x}.hgt.gz
 
 Note: Skadi files are split into 1° by 1° grids. File names refer to the latitude and longitude of the lower left corner of the tile - e.g. N37W105 has its lower left corner at 37 degrees north latitude and 105 degrees west longitude. For example:  N37W105: `https://tile.mapzen.com/mapzen/terrain/v1/skadi/N37/N37W105.hgt.gz?api_key=your-mapzen-api-key`.
 
+## Specify tile size
+
+Both Terrarium and Normal formats support optional tile sizes of `256` and `512` for basic map display, and buffered sizes of `260` and `516` pixels useful for 3d and analytical applications. When not provided, the size defaults to `256`. Historically, the first web slippy maps were based on 256 pixel sized tiles.
+
+```
+https://tile.mapzen.com/mapzen/terrain/v1/{tilesize}/{format}/{z}/{x}/{y}.{extension}?api_key=your-mapzen-api-key
+```
+
+**Larger 512 pixel sized tiles offers several benefits:**
+
+- **Less tiles, less network requests:** a single 512 request is equivalent to four 256 requests
+- **Smaller overall file sizes:** A larger 512 pixel tile compresses to a smaller file size than when split into four 256 tiles
+- **Offline:** Less 512 tiles are needed to cover the same geographic area, and take up less disk space
+
+**Buffered 260 and 516 pixel sized tiles offers several benefits:**
+
+- **Less network requests:** The 2 pixel edge buffers limit reduce network requests from 9 to 1. And a single a single 516 request is equivalent to four 260 requests.
```

Comment:
Might suggest a bit of clarification here: "The 2 pixel edge buffers reduce network requests from 9 to 1, as neighboring tiles no longer need to be sampled."